healthz/checks/checks.go

```diff
@@ -25,29 +25,38 @@ import (
 
 	"github.com/gravitational/satellite/agent/health"
 	pb "github.com/gravitational/satellite/agent/proto/agentpb"
-	"github.com/gravitational/satellite/healthz/config"
 	"github.com/gravitational/satellite/monitoring"
 )
 
-// RunAll runs all checks successively and reports general cluster status
-func RunAll(cfg config.Config) (*pb.Probe, error) {
-	var probes health.Probes
-	var checkers health.Checkers
+// Runner stores configures checkers and provides interface to configure
```

Comment:
s/configures/configured
`Runner configures and execute all available checkers as a group`